samples/matter/common/src/bridge/ble_bridged_device.h

```diff
@@ -76,8 +77,35 @@ class BLEBridgedDeviceProvider : public BridgedDeviceDataProvider {
 
 	bt_addr_le_t GetBtAddress() { return mDevice.mAddr; }
 
+	/**
+	 * @brief Get a number of failed recovery attempts for this provider.
+	 */
+	uint16_t GetFailedRecoveryAttempts() { return mFailedRecoveryAttempts; }
+
+	/**
+	 * @brief Inform provider that recovery attempt for it failed.
+	 *
+	 * This method increments the number of failed recovery attempts.
+	 *
+	 */
+	void NotifyFailedRecovery()
+	{
+		if (mFailedRecoveryAttempts < UINT16_MAX) {
+			mFailedRecoveryAttempts++;
+		}
+	}
+
+	/**
+	 * @brief Inform provider that recovery attempt for it succeeded.
+	 *
+	 * This method resets the number of failed recovery attempts.
+	 *
+	 */
+	void NotifySuccessfulRecovery() { mFailedRecoveryAttempts = 0; }
```

Comment:
The same here. I don't see any notification/callback here. What about renaming it to `ResetRecoveryAttempts`?